bookwyrm/management/commands/generate_preview_images.py

```diff
@@ -0,0 +1,49 @@
+""" Generate preview images """
+import sys
+
+from django.core.management.base import BaseCommand
+
+from bookwyrm import activitystreams, models, settings, preview_images
+
+
+def generate_preview_images():
+    """generate preview images"""
+    print("   | Hello! I will be generating preview images for your instance.")
+    print(
+        "🧑‍🎨 ⎨ This might take quite long if your instance has a lot of books and users."
+    )
+    print("   | ✧ Thank you for your patience ✧")
+
+    # Site
+    sys.stdout.write("   → Site preview image: ")
+    preview_images.generate_site_preview_image_task()
+    sys.stdout.write(" OK 🖼\n")
+
+    # Users
+    users = models.User.objects.filter(
+        local=True,
+        is_active=True,
+    )
+    sys.stdout.write("   → User preview images ({}): ".format(len(users)))
+    for user in users:
+        preview_images.generate_user_preview_image_task(user.id)
```

Comment:
The tasks are not passed with `delay()`. It might be unwise, especially on big instances (like bookwyrm.social), it took some time to generate images for 123 books on my dev environment. Sending them to Celery might be best, what do you think?

Reply:
I think that's a good idea, this would probably stall out on b.s